packages/mattermost-redux/src/types/cloud.ts

```diff
@@ -34,6 +34,8 @@ export type Product = {
     add_ons: AddOn[];
     product_family: string;
     sku: string;
+    billing_scheme: 'flat_fee' | 'per_seat';
+    recurring_interval: 'month' | 'year';
```

Comment:
Is there a related server PR to add this field to the Subscription model? That's why it's probably not showing as `Flat Fee` in the PR test server

Reply:
sure! https://github.com/mattermost/mattermost-server/pull/17874

Reply:
Thanks @pablovelezvidal! If the server changes has the same branch name I believe it will pick up the changes in this PR test server. You can try it out by removing and readding the `Setup Cloud + CWS Test Server`